application/di/test_init.go

```diff
@@ -54,25 +54,25 @@ func TestInit(t *testing.T) {
 	// we don't want to open browsers when testing
 	snyk.DefaultOpenBrowserFunc = func(url string) {}
 	notifier = domainNotify.NewNotifier()
-	analytics = ux.NewTestAnalytics()
+	analytics = ux.NewTestAnalytics(c)
 	instrumentor = performance.NewInstrumentor()
 	errorReporter = er.NewTestErrorReporter()
 	installer = install.NewFakeInstaller()
-	authProvider := snyk.NewFakeCliAuthenticationProvider()
+	authProvider := snyk.NewFakeCliAuthenticationProvider(c)
 	snykApiClient = &snyk_api.FakeApiClient{CodeEnabled: true}
-	authenticationService = snyk.NewAuthenticationService(authProvider, analytics, errorReporter, notifier)
-	snykCli := cli.NewExecutor(authenticationService, errorReporter, analytics, notifier)
+	authenticationService = snyk.NewAuthenticationService(c, authProvider, analytics, errorReporter, notifier)
+	snykCli := cli.NewExecutor(c, authenticationService, errorReporter, analytics, notifier)
 	cliInitializer = cli.NewInitializer(errorReporter, installer, notifier, snykCli)
-	authInitializer := cliauth.NewInitializer(authenticationService, errorReporter, analytics, notifier)
+	authInitializer := cliauth.NewInitializer(nil, authenticationService, errorReporter, analytics, notifier)
```

Comment:
why is the config nil here, is it irrelevant?

Reply:
Nope, that's a bug ;)